fudge-ratio/src/main/webapp/about/about.css

```diff
@@ -161,44 +188,5 @@ h1 {
   color: rgb(37, 51, 51); 
 }
 
-@media only screen and (max-width: 1100px) {
-  #logo {
-    position:relative;  
-    font-size: 10vw;
-    bottom:1vh;  
-  }
-  .bioContainer {
-    position: inherit;
-    padding: 2vh 3vw;
-   }
-  .banner {           
-    width: 100%;
-    height: 10vh;
-    padding-top:5%; 
-  }
-  #title {
-    font-size: 7vh;
-  }
-  .textBox {  
-    margin-top: 5vh;
-  }
-  .row {
-    border-radius: 5vw;
-    position: relative;
-    width: 65vw;
-    display: block;
-    overflow:display;
-    justify-content: center;
-    background: none;
-    margin: 0 auto;
-  }
-  .pictureBox {
-    width: 65vw;
-    height: 65vh;
-  }
-  #giBio, #oscarBio , #jerryBio, #yaaBio {
-    position: inherit;
-    width: 60vw;
-    height: 50vh;
-  } 
-}
```

Comment:
I got rid of this because I made a completely separate CSS file for the mobile version to keep things cleaner